rules/matchers/matchers.py

```diff
@@ -51,3 +51,38 @@ def user_login(cls, rec):
             int(rec['columns']['type']) == cls._EVENT_TYPE_LOGIN and
             (rec['columns']['username'] not in cls._RUNLEVELS)
         )
+
+
+class ConfigMatcher:
```

Comment:
Suggestion, rename to `AwsConfigMatcher` to be slightly more verbose

Reply:
Will do 👌